code/service/src/main/resources/application.conf

```diff
@@ -40,12 +40,20 @@ akka {
 		provider = cluster
 		serializers {
 			proto = "akka.remote.serialization.ProtobufSerializer"
-			cmdSerializer = "com.namely.chiefofstate.CommandSerializer"
+			jackson-cbor = "akka.serialization.jackson.JacksonCborSerializer"
```

Comment:
lets just write a central COS proto serializer and use it. I like the simplicity/control of knowing our serialization is all protobuf today, and i'm willing to help write it.

I personally dislike using language-specific binary serialization.

Reply:
Yeah we can adopt the sole app serializer approach which is cool since we are proto-baseed.